uffizzi/nginx/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM nginx:1.17-alpine
+
+RUN rm /etc/nginx/conf.d/default.conf
+
+COPY ./ /var/www
+COPY ./uffizzi/nginx/nginx /etc/nginx/conf.d/
```

Comment:
I wonder if Uffizzi's new "non-empty" volume feature can do this so you don't have to build a new image?

Reply:
Unfortunately, I think the `non-empty` volumes will not help here, because besides the `conf.d` there is also a copying the of application directory, in default `docker-compose.yml`.

https://github.com/crater-invoice/crater/blob/bc8f2cd4840e42d78682b6e96eb39a26d8b981fe/docker-compose.yml#L44-L45
```
volumes:
      - ./:/var/www
      - ./docker-compose/nginx:/etc/nginx/conf.d/
```